app/lib/import_export.rb

```diff
@@ -379,6 +385,19 @@ def extract_laboratories_details(laboratories, fields)
     laboratories.pluck(*(fields & LABORATORY_FIELD_NAMES.keys))
   end
 
+  # Extract vaccine data values given relevant fields
+  def extract_vaccines_details(vaccines, fields)
+    # join patients if any alternative identifiers are selected
+    vaccines = vaccines.joins(:patient) if (fields & PATIENT_FIELD_TYPES[:alternative_identifiers]).any?
+
+    # validate and pluck selected fields
+    records = vaccines.pluck(*(fields & VACCINE_FIELD_NAMES.keys))
+
+    # remove formula start for 'notes' field
+    notes_index = fields.index(:notes)
```

Comment:
if `notes` is not in `fields`, `notes_index` will be `nil` and `values[notes_index]` will be problematic

can we copy how this is done in other places? https://github.com/SaraAlert/SaraAlert/blob/master/app/lib/import_export.rb#L367